app/assets/stylesheets/v3/layout_components/side_nav.scss

```diff
@@ -111,6 +112,22 @@
         display: block;
       }
     }
+
+    &:nth-of-type(1),
+    &:nth-of-type(2),
+    &:nth-of-type(3),
+    &:nth-of-type(4) {
+      .side-nav__child-list {
+        left: -20px;
+      }
+    }
+
+    &:nth-of-type(5),
+    &:nth-of-type(6) {
```

Comment:
Same for this.
Instead of both selectors, it can be written like this
`&:nth-of-type(n+5) {`